ros_buildfarm/release_job.py

```diff
@@ -699,6 +699,7 @@ def _get_binarydeb_job_config(
         'append_timestamp': build_file.abi_incompatibility_assumed,
 
         'binarydeb_files': binarydeb_files,
+        'build_environment_variables': ['%s=%s' % (var, value) for var, value in build_file.build_environment_variables.items()],
```

Comment:
Again I'm not sure what the preferred style for breaking up this line would be.

Reply:
Commonly we wrap after the open parenthesis / bracket and then where necessary / there is a logical separation, e.g.:

```
        'build_environment_variables': [
            '%s=%s' % (var, value)
            for var, value in build_file.build_environment_variables.items()],
```